expression/builtin_compare_vec.go

```diff
@@ -590,11 +592,113 @@ func (b *builtinNullEQStringSig) vecEvalInt(input *chunk.Chunk, result *chunk.Co
 }
 
 func (b *builtinLTIntSig) vectorized() bool {
-	return false
+	return true
 }
 
 func (b *builtinLTIntSig) vecEvalInt(input *chunk.Chunk, result *chunk.Column) error {
-	return errors.Errorf("not implemented")
+	n := input.NumRows()
+	var err error
+	var buf0, buf1 *chunk.Column
+	buf0, err = b.bufAllocator.get(types.ETInt, n)
+	if err != nil {
+		return err
+	}
+	defer b.bufAllocator.put(buf0)
+	if err := b.args[0].VecEvalString(b.ctx, input, buf0); err != nil {
+		return err
+	}
+	buf1, err = b.bufAllocator.get(types.ETInt, n)
+	if err != nil {
+		return err
+	}
+	defer b.bufAllocator.put(buf1)
+	if err := b.args[1].VecEvalString(b.ctx, input, buf1); err != nil {
+		return err
+	}
+
+	result.ResizeInt64(n, false)
+	result.MergeNulls(buf0, buf1)
+	vecCompareNull(buf0, buf1, result)
+	VecCompareInt(mysql.HasUnsignedFlag(b.args[0].GetType().Flag), mysql.HasUnsignedFlag(b.args[0].GetType().Flag), buf0, buf1, result)
+	vecResOfLT(result)
+	return nil
+}
+
+func vecResOfLT(result *chunk.Column) {
```

Comment:
func vecResOfLT(res []int64)

Reply:
If the incoming parameter is 'res []int64'， in function 'vecResOfLT' we can not distinguish which is null.
![image](https://user-images.githubusercontent.com/13314548/66887031-273b9200-f00c-11e9-9cc0-ccf8963c8e37.png)